docs/man_pages/project/testing/build.md

```diff
@@ -20,24 +26,21 @@ Usage | Synopsis
 * `android` - Build the project for Android and produces an `APK` that you can manually deploy on a device or in the native emulator.
 * `ios` - Build the project for iOS and produces an `APP` or `IPA` that you can manually deploy in the iOS Simulator or on a device.<% } %>
 
-<% if(isHtml) { %>
-
 ### Options
 
 * `--justlaunch` - If set, does not print the application output in the console.
-* `--release` - If set, produces a release build. Otherwise, produces a debug build.
+* `--release` -If set, produces a release build by running webpack in production mode and native build in release mode. Otherwise, produces a debug build. When set, you must also specify the --key-store-* options.
```

Comment:
`When set, you must also specify the --key-store-* options.` - this is not relevant here, it is only for `build android` command